BasePolicies/Governance.md

```diff
@@ -0,0 +1,149 @@
+## Working Group Governance Template
+
+Every Working Group must have a documented governance model that describes the
+basic operation of the Working Group. This must be documented in the repository
+and referenced by the `README.md`.
+
+## Template: *[insert WG name]* Working Group Governance
```

Comment:
Just for clarification, this is CONTRIBUTING.md, correct? Would it make sense to explicitly mention this template goes in CONTRIBUTING.md?